tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -172,3 +191,27 @@ Check GPU Resources
         Page Should Not Contain Element     //a[@href="https://github.com/ContinuumIO/gtc2018-numba"]
     END
 
+Delete Multiple Builds
+    [Documentation]     Deletes Multiple Builds
+    [Arguments]     ${builds_to_be_deleted}
+    FOR    ${build}    IN    @{builds_to_be deleted}
+        ${build_name}=  Search Last Build  namespace=redhat-ods-applications    build_name_includes=${build}
+        Delete Build    namespace=redhat-ods-applications    build_name=${build_name}
+    END
+
+Verify Notification Saying Notebook Builds Not Started
+    [Documentation]     Verifies RHODS Notification Drawer Contains Names of Image Builds which have not started
+    [Arguments]     ${images}
+    RHODS Notification Drawer Should Contain    message=These notebook image builds have not started:
+    FOR    ${image}    IN    @{images}
+        RHODS Notification Drawer Should Contain    message=${image}
+    END
+
+Start Remaining Builds And Close Browser
+    [Documentation]     Starts Remaining builds and closes browser
+    ${builds_to_be_created}=  Create List  s2i-pytorch-gpu-cuda-11.4.2-notebook  s2i-tensorflow-gpu-cuda-11.4.2-notebook
+    FOR    ${build_config}    IN    @{builds_to_be_created}
+        ${build_name}=  Start New Build    namespace=redhat-ods-applications    buildconfig=${build_config}
+        Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${build_name}  expected_status=Complete
```

Comment:
pytorch and tensorflow can run in parallel so you can avoid to check the build status here, and you could check it once both have started

Reply:
Updated.

Reply:
hm I don't think now the test runs tensorflow and pytoch in parallel, because once it starts a build it waits until completion, so it's not parallel. 
It may be complex implement a logic for running in parallel the builds here because we shoudl consider the dependencies..I think it's not worth doing it now. @lugi0 @jgarciao @tarukumar @pablofelix @mwaykole thoughts?

Reply:
I think it's better to keep it simple, even if it takes longer. Also, we'll have pre-built images in the near future